File: MultiplicarMatrices.c
#include <stdio.h>

#define N 3

int main() {
    int A[N][N], B[N][N], C[N][N];
    int i, j, k;

    printf("Ingrese la primera matriz (3x3):\n");
    for(i=0;i<N;i++)
        for(j=0;j<N;j++)
            scanf("%d",&A[i][j]);

    printf("Ingrese la segunda matriz (3x3):\n");
    for(i=0;i<N;i++)
        for(j=0;j<N;j++)
            scanf("%d",&B[i][j]);

    for(i=0;i<N;i++)
        for(j=0;j<N;j++){
            C[i][j]=0;
            for(k=0;k<N;k++)
                C[i][j]+=A[i][k]*B[k][j];
        }

    printf("Resultado de la multiplicacion:\n");
    for(i=0;i<N;i++){
        for(j=0;j<N;j++)
            printf("%6d", C[i][j]);
        printf("\n");
    }

    return 0;
}

File: MultiplicarMatrices.pas
program MultiplicarMatrices;
uses crt;
const n = 3;
var
   A, B, C: array[1..n,1..n] of integer;
   i, j, k: integer;
begin
   clrscr;

   writeln('Ingrese la primera matriz (3x3):');
   for i:=1 to n do
      for j:=1 to n do
         readln(A[i,j]);

   writeln('Ingrese la segunda matriz (3x3):');
   for i:=1 to n do
      for j:=1 to n do
         readln(B[i,j]);

   for i:=1 to n do
      for j:=1 to n do
      begin
         C[i,j] := 0;
         for k:=1 to n do
            C[i,j] := C[i,j] + A[i,k] * B[k,j];
      end;

   writeln('Resultado de la multiplicacion:');
   for i:=1 to n do
   begin
      for j:=1 to n do
         write(C[i,j]:6);
      writeln;
   end;

   readln;
end.
